contracts/FlowIDTableStaking.cdc

```diff
@@ -1196,10 +1206,22 @@ pub contract FlowIDTableStaking {
                     // No action is needed to mark the nodes as added because they are already included
                     currentNodeCount[role] = currentNodeCount[role]! + UInt16(candidateNodesForRole.keys.length)
                 }
+
+                nodesToAdd.appendAll(candidateNodesForRole.keys)
+
+                // Add the desired open slots for each role to the current node count
+                // to make the slot limits
+                if let openSlotsForRole = openSlots[role] {
+                    slotLimits[role] = currentNodeCount[role]! + openSlotsForRole
```

Comment:
We could _lower_ the slot limit for a role here, which I don't think is the desired behaviour. 

For example, suppose `slotLimits[r]=100`, `currentNodeCount[r]=50`, and `openSlots[role]=10`. Then we would end up setting `slotLimits[r]=60`.

Reply:
good catch. So we could just check to see if the calculated slot slot limits is less than the current slot limits and only set it if it is greater, but I could potentially see this behavior being desirable if we decide we want less slot limits than what we currently have

Reply:
> I could potentially see this behavior being desirable if we decide we want less slot limits than what we currently have

Yeah, I'm not completely sure which behaviour we want. If we allow the `openSlots` process to _remove_ slots though, I feel we should document it in the parts of code dealing with `slotLimits` (eg. [here](https://github.com/onflow/flow-core-contracts/blob/access-slots/transactions/idTableStaking/admin/set_slot_limits.cdc#L3-L8))

Reply:
documented in the latest commit